lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/Jordan_Eldridge/AnimalTest.java

```diff
@@ -0,0 +1,40 @@
+package com.codedifferently.lesson7.Jordan_Eldridge;
```

Comment:
Prefer package name that is lowercase alphanumeric, update the file path accordingly.